weave-js/src/components/PagePanelComponents/Home/Browse2/RunsTable.tsx

```diff
@@ -128,7 +143,7 @@ export const RunsTable: FC<{
           _.mapKeys(
             _.omitBy(args, v => v == null),
             (v, k) => {
-              return 'input_' + k;
+              return 'input.' + k;
```

Comment:
For consistency with attributes and output, didn't appear to be a reason for it to be different.